database/G/George Thorogood And The Destroyers/Maverick/I Drink Alone

```diff
@@ -0,0 +1,54 @@
+I drink alone
+Ye-yeah, with nobody else
+I drink alone
+Ye-yeah, with nobody else
+Yeah, you know when I drink alone
+I prefer to be by myself
+
+Now every morning just before breakfast
+I don’t want no coffee or tea
+Just me and good buddy Wiser
+That’s all I ever need
+’Cause I drink alone, yeah
+With nobody else
+Yeah, you know when I drink alone
+I prefer to be by myself
+
+Yeah, the other night I laid sleeping
+And I woke from a terrible dream
+So I caught up my pal Jack Daniel
+And his partner Jimmy Beam
+And we drank alone
+Ye-yeah, with nobody else
+Yeah, you know when I drink alone
+I prefer to be by myself
+
+Yeah, the other day I got invited to a party
+But I stayed home instead
+Just me and my pal Johnnie Walker
+And his brothers Black and Red
+And we drank alone
+Ye-yeah, with nobody else
+Yeah, you know when I drink alone
+I prefer to be by myself
+
+Yeah, my whole family done give up on me
```

Comment:
"Done give up" sounds strange, is it an imitation of some dialect?

Reply:
It may be, I think I need to listen to it more closely, with headphones this time.

Reply:
Seems to be "Yeah, my whole family done give up on me." on the LP's lyric book.